analytic_operating_unit/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
```

Comment:
@sudhir-erpharbor IMO remove `Copyright` from all `__init__` file.

Reply:
@aheficent I am not sure about this as I saw many OCA modules in v12 that have such copy rights. Please guide us in this.

Reply:
@sudhir-erpharbor Can you please [see](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py)